test/e2e/mmi/pageObjects/mmi-accountMenu-page.ts

```diff
@@ -99,9 +99,17 @@ export class MMIAccountMenuPage {
     const dialog = this.page
       .getByRole('dialog')
       .filter({ hasText: 'Select an account' });
+
+    const bannerCloseButton = dialog.locator('.mm-banner-base__close-button');
+
+    if ((await bannerCloseButton.count()) > 0) {
+      await bannerCloseButton.click();
+    }
```

Comment:
This code closes the banner so that it matches with the screenshots on the tests.

Reply:
would it work something like, I think this will wait until the locator is found or timeout
await dialog.locator(....).click();

Reply:
It can be done, but I had to refactor more things in order to achieve it.

Once you close the banner, it won't show again if you reopen the accounts menu list.

The test on `test/e2e/mmi/specs/visual.spec.ts` takes two screenshots, every a screenshot is taken, `accountMenuScreenshot` first opens the menu, then closes the banner and finally it takes the screenshot. That means you need to account for the fact that the banner might not be visible.

So what I had to do in order not to check that is to remove both the action to open the dialog and to close the banner from `accountMenuScreenshot` and call them from the test itself.

Either solution should be fine.

Reply:
Happy with the change. I thought this was going to affect other tests, but I see that we are only screenshot checking this test! cool!